.github/workflows/tests.yml

```diff
@@ -7,19 +7,39 @@ on:
   pull_request:
 
 jobs:
-  build:
+  unit:
+    timeout-minutes: 10
     runs-on: ubuntu-latest
     name: Unit Tests
     steps:
       - name: Checkout Commit
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
       - name: Setup Node
-        uses: actions/setup-node@v2
+        uses: actions/setup-node@v3
         with:
-          node-version: '16'
+          node-version: 16
       - name: Install dependencies
-        run: npm ci && cd frontend && npm ci
+        run: npm ci
       - name: Run tests
-        run: cd frontend && npm test
+        run: cd frontend && npm test && cd ..
```

Comment:
How is this working? `package.json` is located at the root level. If I would run `cd frontend && npm test && cd ..` locally, it would fail.

Reply:
That is a very good question indeed...